databases/week3/meal_sharing.sql

```diff
@@ -0,0 +1,196 @@
+SET NAMES utf8mb4;
+
+CREATE DATABASE meal_sharing;
+USE meal_sharing;
+
+ --  Create mealsharing database
+ CREATE TABLE `meal`(
+`id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`title` VARCHAR(255) NOT NULL,
+`description` TEXT NOT NULL,
+`location` VARCHAR(255) NOT NULL,
+`when` DATETIME NOT NULL,
+`max_reservations` INT UNSIGNED NOT NULL,
+`price` DECIMAL(8,2) NOT NULL,
+`created_date` TIMESTAMP
```

Comment:
Things like created and updated columns are places where you could argue a `timestamp` might be more appropriate than `datetime`. However in the reservation and review tables you are using `datetime`. Don't use different data types for the same thing in different tables